src/main/java/com/google/cloud/spanner/pgadapter/statements/MatcherStatement.java

```diff
@@ -55,7 +55,10 @@ private String translateSQL(String sql) {
     for (Command currentCommand :
         Command.getCommands(sql, this.connection, this.commandMetadataJSON)) {
       if (currentCommand.is()) {
-        return currentCommand.translate();
+        System.out.println("DEBUG: ");
```

Comment:
I don't think these debug statements should be here.

Reply:
Looks like in PG, hashing is not used. `SELECT a.attname,
  pg_catalog.format_type(a.atttypid, a.atttypmod),
  (SELECT pg_catalog.pg_get_expr(d.adbin, d.adrelid, true)
   FROM pg_catalog.pg_attrdef d
   WHERE d.adrelid = a.attrelid AND d.adnum = a.attnum AND a.atthasdef),
  a.attnotnull,
  (SELECT c.collname FROM pg_catalog.pg_collation c, pg_catalog.pg_type t
   WHERE c.oid = a.attcollation AND t.oid = a.atttypid AND a.attcollation <> t.typcollation) AS attcollation,
  a.attidentity,
  a.attgenerated
FROM pg_catalog.pg_attribute a
WHERE a.attrelid = '16385' AND a.attnum > 0 AND NOT a.attisdropped
ORDER BY a.attnum;`
check a.attrelid = '16385' here.

If there is any reason a hashing fucntion is needed we can add it back once the backend issue is fixed. This is just a quick patch.